src/geoschem_run_scripts/submit_jacobian_simulations_array.sh

```diff
@@ -4,7 +4,7 @@ echo "running {END} jacobian simulations" >> {InversionPath}/imi_output.log
 # remove error status file if present
 rm -f .error_status_file.txt
 
-sbatch --array={START}-{END} --mem $JacobianMemory \
+sbatch --array={START}-{END}{JOBS} --mem $JacobianMemory \
```

Comment:
What does {JOBS} do here?

Reply:
@laestrada Just seeing this question now. In the configuration file, there is a new option `MaxSimultaneousRuns` that allows users to submit their Jacobian runs in batches rather than submitting them all to the queue at once. This is especially useful if the number of state vector elements (and therefore Jacobian runs) is large. If `MaxSimultaneousRuns` is greater than zero then `{JOBS}` will be replaced with `%` followed by the number of runs specified. If `MaxSimutaneousRuns` is -1 (or really any value less than zero) then `{JOBS}` will be removed from that line and the entire job array will be submitted to the queue at once.